setup.cfg

```diff
@@ -16,10 +16,7 @@ keywords = wrapper, proxy, decorator
 classifiers =
     Development Status :: 5 - Production/Stable
     License :: OSI Approved :: BSD License
-    Programming Language :: Python :: 2
-    Programming Language :: Python :: 2.7
```

Comment:
Given that you drop support for Python 2.7 explicitly, I guess it's perfectly valid to release it as 1.14.0 (contrary to what I originally wrote in https://github.com/GrahamDumpleton/wrapt/issues/189#issuecomment-936164339). I realized that pip will not install 1.14.0 in a Python 2.7 environment if the support is dropped explicitly.

Thanks a lot for this change ❤ - I fully support it (but you do not have to listen to a n00b like me 😉)